java/test/jmri/jmrix/marklin/cdb/serialdriver/configurexml/ConnectionConfigXmlTest.java

```diff
@@ -0,0 +1,43 @@
+package jmri.jmrix.marklin.cdb.serialdriver.configurexml;
+
+import jmri.util.JUnitUtil;
+
+import org.junit.jupiter.api.*;
+ 
+/**
+ * Tests for the ConnectionConfigXml class.
+ * @author Steve Young Copyright (C) 2024
+ */
+public class ConnectionConfigXmlTest extends jmri.jmrix.configurexml.AbstractSerialConnectionConfigXmlTestBase {
+
+    @Test
+    @Override
+    public void storeTest() {
+        super.storeTest();
+        jmri.util.JUnitAppender.suppressErrorMessage("No usable ports returned");
+    }
+
+    @Test
+    @Override
+    public void loadTest() {
+        super.storeTest();
```

Comment:
This doesn't look right. Should it be super.loadTest() instead?

Reply:
Thanks, good catch :-)